plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1664,6 +1664,29 @@ private void testCreateTableWithCommentsForColumnMappingMode(ColumnMappingMode m
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
```

Comment:
follow-up let's add testing resources similar to `plugin/trino-delta-lake/src/test/resources/deltalake/partition_values_parsed` for column mapping mode `id` and `name`